client/plots/matrix.interactivity.js

```diff
@@ -256,6 +257,64 @@ export function setInteractivity(self) {
 					self.dom.clickMenu.d.selectAll('*').remove()
 				})
 		}
+
+		const showHnEImages = JSON.parse(sessionStorage.getItem('optionalFeatures') || `{}`)?.showHnEImages
+		if (q.HnEImages && showHnEImages) {
```

Comment:
just a note H&E is one type of such images. there are other belonging to deep zoom category. later the datatype could be renamed to `ds.queries.DZimage{}` with a `imageType='H&E'`, and on client shows q.DZimage.imageType in place of hardcoded `H&E` strings